ci/scripts/docs_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+arrow_dir=${1}
+build_dir=${2}
+
+export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
+export PKG_CONFIG_PATH=${ARROW_HOME}/lib/pkgconfig:${PKG_CONFIG_PATH}
+export CXXFLAGS="-D_GLIBCXX_USE_CXX11_ABI=0"
+
+# Sphinx
+sphinx-build -b html ${arrow_dir}/docs/source ${arrow_dir}/docs/_build
+
+# C++
+pushd ${arrow_dir}/cpp/apidoc
+doxygen
+popd
+
+# C GLib
+# pushd ${arrow_dir}/c_glib
+# if [ -f Makefile ]; then
+#     # Ensure updating to prevent auto re-configure
+#     touch configure **/Makefile
+#     make distclean
+# fi
+# ./autogen.sh
+# mkdir -p apidoc
+# pushd apidoc
+# ../configure --prefix=${ARROW_HOME} --enable-gtk-doc
```

Comment:
It is missing glib related pc files, not sure why. I've tried to explicitly add to PKG_CONFIG_PATH but it didn't work. I probably miss something. @kou could you help me with it?

Reply:
After uncommenting the following commands should reproduce the issue:
```
docker-compose build conda-cpp
docker-compose build conda-python
docker-compose build conda-docs
docker-compose run --rm conda-docs
```

Reply:
Conda based image isn't suitable to build Apache Arrow GLib document because conda based image doesn't include GLib.

Debian 10 or Ubuntu 18.04 based image will be better.